### PR TITLE
#7744

### PR DESCRIPTION
**Pull request by HeorhiiBoliuk**
Created a drush command that changes text format for article on basic html, and hook_post_update_NAME that changes text format on limited HTML.

### CODE REVIEW

**Comment by lavrenchuk-dev** [comment]
> sfjodsof

dsfdsf

**Review by ivandudardb — APPROVED** [review]
LGTM

File: web/modules/custom/batch_api_drush/src/Commands/BatchApiDrushCommands.php [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+use Drupal\node\Entity\Node;
+use Drupal\node\NodeInterface;
+use Drush\Commands\DrushCommands;
+
+/**
+ * A Drush commandfile.
+ *
+ * In addition to this file, you need a drush.services.yml
+ * in root of your module, and a composer.json file that provides the name
+ * of the services file to use.
```
Comment (lavrenchuk-dev) [inline web/modules/custom/batch_api_drush/src/Commands/BatchApiDrushCommands.php:19]: Not sure about this comment

File: web/modules/custom/batch_api_drush/src/Commands/BatchApiDrushCommands.php [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+   */
+  public function finished($success, $results, $operations): void {
+    $message = $this->t('Number of nodes affected by batch: @processed', ['@processed' => $results['processed']]);
+    $this->messenger->addStatus($message);
+  }
+
+  /**
+   * Get all nodes with type 'article'.
+   */
+  public function getNodes(): int|array {
```
Comment (lavrenchuk-dev) [inline web/modules/custom/batch_api_drush/src/Commands/BatchApiDrushCommands.php:125]: Does this need to be UNION type?

File: web/modules/custom/batch_api_drush/src/Commands/BatchApiDrushCommands.php [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    return $this->entityTypeManager->getStorage('node')
+      ->getQuery()
+      ->condition('status', NodeInterface::PUBLISHED)
+      ->condition('type', 'article')
+      ->accessCheck(FALSE)
+      ->execute();
```
Comment (lavrenchuk-dev) [inline web/modules/custom/batch_api_drush/src/Commands/BatchApiDrushCommands.php:131]: The whole purpose of batch API in that case is to not try to load all entities.
Let's imagine you have 1 million entities that need to be updated in any way
You'll never be able to load all of them at once

File: web/modules/custom/batch_limited_html/batch_limited_html.post_update.php [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+ * Get all nodes.
+ */
+function get_nodes() {
+  $query = \Drupal::entityQuery('node');
+  $query->condition('status', NodeInterface::PUBLISHED);
+  $query->condition('type', 'article');
+  $query->accessCheck(FALSE);
+  $nids = $query->execute();
+
+  return Node::loadMultiple($nids);
```
Comment (lavrenchuk-dev) [inline web/modules/custom/batch_limited_html/batch_limited_html.post_update.php:71]: The whole purpose of batch API in that case is to not try to load all entities.
Let's imagine you have 1 million entities that need to be updated in any way
You'll never be able to load all of them at once

**Review by lavrenchuk-dev — CHANGES_REQUESTED** [review]

**Review by lavrenchuk-dev — APPROVED** [review]

**Pull request merged**